keyboards/keebio/quefrency/keymaps/bfiedler/keymap.c

```diff
@@ -0,0 +1,77 @@
+/* Copyright 2021 Ben Fiedler
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// https://beta.docs.qmk.fm/using-qmk/software-features/tap_hold
+#define TAPPING_TERM 120
+#define IGNORE_MOD_TAP_INTERRUPT
+#define PERMISSIVE_HOLD
+
+// This does not disable RGB entirely, but it disables it by default.
+#undef RGBLIGHT_ENABLE
+
+#include QMK_KEYBOARD_H
+
+#define KC_RST RESET
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BASE 0
+#define _FN1 1
```

Comment:
We heavily recommend enums, as they're easier to maintain, long term.
```suggestion
enum custom_layers {
    _BASE,
    _FN1,
};    
```